client/tx/tx.go

```diff
@@ -344,8 +345,9 @@ func SignWithPrivKey(
 	}
 
 	sigV2 = signing.SignatureV2{
-		PubKey: priv.PubKey(),
-		Data:   &sigData,
+		PubKey:          priv.PubKey(),
+		Data:            &sigData,
+		AccountSequence: accSeq,
```

Comment:
In all other places I encountered to far, the fields were callen "account number" and "sequence". I think it is consistent to drop "Account" and you don't loose anything while saving a lot of text.

Reply:
Thank you for this change. Makes everyone's life easier. The term "account sequence" seems to be introduced in ADR 020 by accident rather than for an explicit reason.

Reply:
Apparently they also use `AccountSequence` in `authsigning.SignerData`. So, the tx/signing stuff is (relatively) consistently using `AccountSequence`, while `BaseAccount` is using `Sequence`. They should definitley stick to a consistent naming. (As you read one to write the other)

Reply:
@amaurymartiny I wouldn't block the PR on naming. This may need other sdk discussions, but if not addressed in this PR, please make an issue pointing to this comment and ping me.